profiles/core/inputs.tf

```diff
@@ -578,25 +584,25 @@ variable "stac_fastapi_secret" {
   sensitive   = true
 }
 
-variable stac_namespace {
+variable "stac_namespace" {
   type        = string
   description = "Namespace for STAC-FastAPI"
   default     = "stac"
 }
 
-variable stac_version {
-  type = string
+variable "stac_version" {
+  type        = string
   description = "Version of STAC-FastAPI Helm Chart"
-  default = "0.1.0"
+  default     = "0.1.0"
 }
 
-variable stac_fastapi_additional_configuration_values {
+variable "stac_fastapi_additional_configuration_values" {
   type        = list(string)
   default     = []
   description = "List of values in raw yaml to pass to helm. Values will be merged, in order, as Helm does with multiple -f options."
 }
```

Comment:
were missing the keys and region inputs

```
variable "aws_access_key" {
  description = "AWS Access Key ID"
  type        = string
  default     = ""
  sensitive   = true
}

variable "aws_secret_access_key" {
  description = "AWS Secret Access Key"
  type        = string
  default     = ""
  sensitive   = true
}

variable "aws_region" {
  description = "AWS Region"
  type        = string
  default     = ""
  sensitive   = true
}

```
along with the optional session token

Reply:
we also need to add these at the upper inputs.tf at the top level of the repo in: https://github.com/Element84/filmdrop-k8s-tf-modules/blob/config/inputs.tf